src/java.desktop/share/classes/sun/awt/image/ByteArrayImageSource.java

```diff
@@ -52,10 +51,7 @@ final boolean checkSecurity(Object context, boolean quiet) {
     }
 
     protected ImageDecoder getDecoder() {
-        InputStream is =
-            new BufferedInputStream(new ByteArrayInputStream(imagedata,
-                                                             imageoffset,
-                                                             imagelength));
+        InputStream is = new ByteArrayInputStream(imagedata, imageoffset, imagelength);
```

Comment:
This file use 80 chars per line code style.

Reply:
Fixed.